pkg/nat/nat.go

```diff
@@ -84,6 +102,14 @@ func (p Port) Int() int {
 	return int(port)
 }
 
+// Range returns the start/end port numbers of a Port range as ints
+func (p Port) Range() (int, int) {
+	// We don't need to check for an error because we're going to
+	// assume that any error would have been found, and reported, in NewPort()
+	portStart, portEnd, _ := ParsePortRange(p.Port())
```

Comment:
I agree with @mavenugo, `Range()` is an exported function and `type Port` can be constructed in an unsafe way.